tutorials/Atmos/risingbubble.jl

```diff
@@ -13,10 +13,12 @@
 # ## Description of experiment
 # 1) Dry Rising Bubble (circular potential temperature perturbation)
 # 2) Boundaries
-#    - `Impenetrable(FreeSlip())` - no momentum flux, no mass flux through
+#    Top and Bottom boundaries:
+#    - `Impenetrable(FreeSlip())` - Top and bottom: no momentum flux, no mass flux through
 #      walls.
 #    - `Impermeable()` - non-porous walls, i.e. no diffusive fluxes through
 #       walls.
+#    Lateral boundaries
```

Comment:
If we emphasize that these are default boundary conditions, the line below about Applying boundary conditions(in the scope of tutorial note block) can be removed.